cli/build.gradle.kts

```diff
@@ -21,6 +21,10 @@ dependencies {
   // testImplementation("org.ice1000.jimgui", "fun", version = deps.getProperty("version.jimgui"))
 }
 
+plugins {
+  id("org.graalvm.buildtools.native") version "0.9.11"
```

Comment:
Does this make Aya depend on a graalvm JVM?

Reply:
No. Only the command `gradle nativeCompile` requires a GraalVM installation, and the installation doesn't have to be the Gradle JVM. See new review comments.

Reply:
And our CI on this PR still passes, thus Aya is still buildable on a GraalVM-free machine.

Reply:
Nice